db/migrate/20160601134855_move_existing_withdrawn_notices.rb

```diff
@@ -0,0 +1,22 @@
+class MoveExistingWithdrawnNotices < ActiveRecord::Migration
+  def change
+    Unpublishing.where(type: "withdrawal").each do |unpublishing|
+      content_item = unpublishing.content_item
```

Comment:
How many of these are there?  If lots should we use `find_each` and/or eager load the content item?